src/renderer/screens/platform/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./Catalog";
```

Comment:
Why aren't things inside platform, or we change the screen to be inside catalog instead of this? 🤔